### ISSUE TITLE
Incorrect symbol names displayed in example plot

### ISSUE THREAD

**Issue by michelececcacci** [Davide-sd/sympy-plot-backends#16]
`Python 3.9.13`
`Spb version: 1.6.6`
`Sympy version: 1.10.1`
As you can see in the image i uploaded, In the following example (second in the verifying installation folder) every name has a `$` character prepended and one appended. 
To reproduce, just paste this in an empty python repl interpreter:
```py
from sympy import *
from spb import *
x, a, b, c = symbols("x, a, b, c")
plot(
    (cos(a * x + b) * exp(-c * x), "oscillator"),
    (exp(-c * x), "upper limit", {"line_dash": "dotted"}),
    (-exp(-c * x), "lower limit", {"line_dash": "dotted"}),
    (x, 0, 2 * pi),
    params={
        a: (1, 0, 10),     # frequency
        b: (0, 0, 2 * pi), # phase
        c: (0.25, 0, 1)    # damping
    },
    ylim=(-1.25, 1.25),
    backend=BB,
    servable=True
)
```
![image](https://user-images.githubusercontent.com/75946413/216831725-02b74cfe-5b6b-4920-813c-83cd30381747.png)

**Comment by Davide-sd** [comment]
Yep, known behavior caused by Bokeh version < 3 which didn't fully supported Latex. Holoviz's Panel uses Bokeh to create widgets. The current version of Panel uses Bokeh 2.4.3. Full Latex support is available in Bokeh 3.0 and up, but Panel still has to catch up. Following [Bokeh's doc](https://docs.bokeh.org/en/dev-3.0/docs/user_guide/styling/mathtext.html), once that happens, [this line of code](https://github.com/Davide-sd/sympy-plot-backends/blob/master/spb/interactive.py#L114) should be changed to use double $ wrapper.

**Comment by michelececcacci** [comment]
Ok, perfect. Thanks for the quick reply!
